experiment/measurer.py

```diff
@@ -536,21 +537,30 @@ def generate_profdata(self, cycle: int):
             self.logger.error(
                 'Coverage profdata generation failed for cycle: %d.', cycle)
 
-    def generate_summary(self, cycle: int):
+    def generate_summary(self, cycle: int, summary=True):
         """Transform the .profdata file into json form."""
         coverage_binary = get_coverage_binary(self.benchmark)
         command = [
             'llvm-cov', 'export', '-format=text', coverage_binary,
             '-instr-profile=%s' % self.profdata_file
         ]
+
+        # Add summary-only flag to reduce memory use.
```

Comment:
Remove this comment since it is not needed during trials even, this is bad for perf.